deployment/terraform/static.tf

```diff
@@ -0,0 +1,166 @@
+# Static website
+
+resource "aws_s3_bucket" "marlowe_playground" {
+  bucket = "marlowe-playground-website"
+  acl    = "public-read"
+
+  website {
+    index_document = "index.html"
+    error_document = "error.html"
+  }
+}
+
+resource "aws_s3_bucket_policy" "marlowe_playground_website" {
+  bucket = aws_s3_bucket.marlowe_playground.id
+
+  policy = <<POLICY
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "PublicReadGetObject",
+            "Effect": "Allow",
+            "Principal": "*",
+            "Action": [
+                "s3:GetObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::marlowe-playground-website/*"
+            ]
+        }
+    ]
+}
+POLICY
+}
+
+resource "aws_iam_role" "s3_proxy_role" {
+  name               = "marlowe_website_s3_${var.env}"
+  path               = "/"  
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "apigateway.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
```

Comment:
same answer.

Reply:
plus it might be tricky, not sure how json would look, I don't really think it would look any better unless you extracted it to a file, then there would be a level of indirection